### PR TITLE
Feature/UI 13.6 vg 1.4

### PR DESCRIPTION
**Pull request by gbalan-microej**

### CODE REVIEW

File: VEEPortingGuide/uiReleaseNotes.rst [hunk 1]
```diff
@@ -18,7 +18,9 @@ Standard Versions
 +-----------------+--------------------+---------------------------------------------------------+
 | UI Pack Range   | Architecture Range | Comment                                                 |
 +=================+====================+=========================================================+
-| [13.0.0-13.5.1] | [7.13.0-8.0.0[     | SNI 1.3                                                 |
+| [13.5.0-13.6.0] | [7.13.0-9.0.0[     | Compatibility with Architecture 8                       |
```
Comment (gbalan-microej) [inline VEEPortingGuide/uiReleaseNotes.rst:21]: [13.5.0-13.6.**1**]

File: ApplicationDeveloperGuide/UI/MicroVG/vectorimage.rst [hunk 2]
```diff
@@ -239,6 +239,8 @@ The SVG format also supports the animation of vector graphics objects, but this
 
 SVG files that need to be animated should be converted to Android Vector Drawable format with the Android Vector Asset tool and then animated manually or with a tool like `Shapeshifter <https://shapeshifter.design/>`_.
 
+.. warning:: A bug in Eclipse Temurin™ JDK 8 causes animated vector images to render incorrectly on the simulator. You should upgrade to Eclipse Temurin™ JDK 11 or use the JDK from Oracle instead.
```
Comment (gbalan-microej) [inline ApplicationDeveloperGuide/UI/MicroVG/vectorimage.rst:242]:
```suggestion
.. warning:: A defect in Eclipse Temurin™ JDK 8 causes animated vector images to render incorrectly on the Simulator. You should upgrade to Eclipse Temurin™ JDK 11 or use the JDK from Oracle instead.
```

* a "bug": another word ? issue, defect ?
* Actually all refs in the VG & UI chapters are wrong: the other chapters use "Simulator"

File: VEEPortingGuide/vgChangeLog.rst [hunk 3]
```diff
@@ -4,6 +4,22 @@
 Changelog
 =========
 
+[1.4.0] - (Unreleased)
+======================
+
+XXX TODO XXX
```
Comment (gbalan-microej) [inline VEEPortingGuide/vgChangeLog.rst:10]: to remove

File: VEEPortingGuide/vgChangeLog.rst [hunk 4]
```diff
@@ -4,6 +4,22 @@
...
+
+MicroVG Implementation
+""""""""""""""""""""""
+
+**Changed**
+
+* Compatible with `MicroVG API 1.4`_.
+
+.. _MicroVG API 1.4: https://repository.microej.com/modules/ej/api/microvg/1.4.0/
+
```
Comment (gbalan-microej) [inline VEEPortingGuide/vgChangeLog.rst:22]: somewhere: the system view events (available in CCO, not in pack. TBD)

File: VEEPortingGuide/uiMigrationGuide.rst [hunk 5]
```diff
@@ -6,6 +6,87 @@
 Migration Guide
 ===============
 
+From 13.5.x to 13.6.x
+=====================
+
+Front Panel
+"""""""""""
+
+* Set the explicit dependency to the `UI Pack 13.6.1`_:
```
Comment (gbalan-microej) [inline VEEPortingGuide/uiMigrationGuide.rst:15]: FYI the other chapters don't use a link.

**Review by microej-sh — APPROVED** [review]

**Pull request merged**